UI/PopulationPanel.cpp

```diff
@@ -70,7 +70,30 @@ void PopulationPanel::CompleteConstruction() {
     std::vector<std::pair<MeterType, MeterType>> meters;
 
     for (auto& meter_stat : m_meter_stats) {
-        meter_stat.second->InstallEventFilter(shared_from_this());
+        meter_stat.second->RightClickedSignal.connect([this, &meter_stat](const GG::Pt& pt) {
+            MeterType meter_type = meter_stat.first;
+
+            std::string meter_string = boost::lexical_cast<std::string>(meter_type);
+
+            auto popup = GG::Wnd::Create<CUIPopupMenu>(pt.x, pt.y);
+            auto pc = this->GetPopCenter();
+            if (meter_type == METER_POPULATION && pc) {
+                std::string species_name = pc->SpeciesName();
```

Comment:
Zoom to species doesn't seem to work in my tests. Quickstarted a game, right click the population icon on my planet and get a two-command popup, pick "Pedia Lookup Human" but nothing happens. Other meter pedia lookup commands do work, though.

Reply:
OK, not sure if this issue is in the PopulationPanel or the MultiIconValueIndiactor. It happens when the population panel is NOT expanded, but it works fine when it is expanded.

Reply:
Cannot reproduce.  Seed 0, Race human, quick start game.

Reply:
To illustrate the issue, I'm trying to use this popup command:

![FailedLookupPopup](https://user-images.githubusercontent.com/11386908/71876423-f9f01300-3126-11ea-87d7-e573b89ad0d1.png)

Switching to the de stringtable doesn't change the result.

Reply:
Still not reproducible, still works as expected.